packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js

```diff
@@ -5,6 +5,16 @@ import { exactProp, elementAcceptingRef } from '@material-ui/utils';
 import ownerDocument from '../utils/ownerDocument';
 import useForkRef from '../utils/useForkRef';
 
+const focusSelectorsRoot = [
```

Comment:
We should also consider elements that has `tabindex=0` at least, don't we? It may be harder with elements that has other positive values 🤔